packages/ra-ui-materialui/src/list/Empty.tsx

```diff
@@ -38,15 +42,10 @@ const Empty: FC<EmptyProps> = props => {
     const classes = useStyles(props);
     const translate = useTranslate();
 
+    const getResourceLabel = useGetResourceLabel();
     const resourceName = translate(`resources.${resource}.forcedCaseName`, {
         smart_count: 0,
-        _: inflection.humanize(
-            translate(`resources.${resource}.name`, {
-                smart_count: 0,
```

Comment:
Was it an error? smart_count zero seems wrong to me

Reply:
It's not. See https://airbnb.io/polyglot.js/#pluralization. Some languages have special rules when there no elements at all.